test/integration/targets/template/aliases

```diff
@@ -1,3 +1,4 @@
 needs/root
+needs/target/setup_remote_tmp_dir
```

Comment:
This test also runs on the controller, so shouldn't need to use remote_tmp_dir.

Reply:
reverted